src/my-view1.html

```diff
@@ -193,7 +193,9 @@
     <div class="content">
       <div class="description">
       <h2 class="intro-description center color-secondary">
-        Create an email to request a free bike corral from the City of Austin
+        <strong>Make your voice heard!</strong> 
+        <br>
+        Request Yo Racks is a free, responsive web application that helps you to request bike corrals to be installed directly to the City of Austin via email.
```

Comment:
I like the idea of the elevator speech. However it should not be placed here. It fills all the space on the image and breaks the mobile version of the web app.

Desktop version:
<img width="1027" alt="screen shot 2018-06-28 at 10 28 10 am" src="https://user-images.githubusercontent.com/6969134/42044226-0bb04220-7abe-11e8-92c7-32a3f8830dd8.png">

Mobile version:
![tagline - mobile](https://user-images.githubusercontent.com/6969134/42044272-2ea10b02-7abe-11e8-8b8e-151c2cb290b5.png)